modules/tools/navigation/planning/navigation_routing_debug.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+###############################################################################
+# Copyright 2017 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+import rospy
+import time
+import json
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from std_msgs.msg import String
+from numpy.polynomial.polynomial import polyval
+from modules.drivers.proto import mobileye_pb2
+from modules.planning.proto import planning_pb2
+from modules.canbus.proto import chassis_pb2
+from modules.localization.proto import localization_pb2
+from path_decider import PathDecider
+from speed_decider import SpeedDecider
+from trajectory_generator import TrajectoryGenerator
+from provider_mobileye import MobileyeProvider
+from provider_chassis import ChassisProvider
+from provider_localization import LocalizationProvider
+from provider_routing import RoutingProvider
+
+planning_pub = None
+routing_debug_pub = None
+
+PUB_NODE_NAME = "planning"
+PUB_TOPIC = "/apollo/" + PUB_NODE_NAME
+CRUISE_SPEED = 10  # m/s
```

Comment:
line 43 ~ 45, three variables are defined but not used.

Reply:
first two variables are used, the cruise speed will be used soon.